### DISCUSSION TITLE
elf ears issue

### DISCUSSION THREAD

**Discussion by ghost** [Monika-After-Story/MonikaModDev#7783] [general - spritepacks]
so, i have elf ears, i put it in chokers section, and tried the body section, but it keeps going behind her ears. see?
![Screen Shot 2021-09-10 at 10 21 16 AM](https://user-images.githubusercontent.com/82217573/132877795-da4960c1-e2f1-4541-b167-b40353076c79.png)

**Comment by akloholik** [comment]
did you tried to add it in the hairpin section? it might work better since it would be in front and not behind.

**Comment by ghost** [comment]
> did you tried to add it in the hairpin section? it might work better since it would be in front and not behind.

i cant add it there. it just shows up as a choker.
![Screen Shot 2021-09-18 at 5 43 53 PM](https://user-images.githubusercontent.com/82217573/133910369-8fe3486e-16d9-49ac-95e9-607a645a5db7.png)

**Comment by ThePotatoGuy** [comment]
If this is intended to be like wearable elf ears, then the submod creator should make this into its own selector.
